.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jquery": true
+    },
+    "extends": [
+        "eslint:recommended"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly",
+        "OpenLayers": "readonly",
+        "lizMap": "readonly",
+        "lizUrls": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+        "indent": [
+            "error",
+            4,
+            {
+                "SwitchCase": 1,
+                "ignoredNodes": [
+                    "TemplateLiteral *"
+                ]
+            }
+        ]
+    }
+}
```

Comment:
It should be an empty line at the end of a file.

Reply:
@drillinP auto empty line can be configured in vscode.